Blazor/README.md

```diff
@@ -125,12 +123,19 @@ To use, in your project:
     ...    
 ```
   - If using the application template, simply use `appTop` and `appFooter` instead
+  - When calling InstallCDTS in Program.cs, one of the optional parameters is a boolean that needs to be set to true if using the application template
```

Comment:
This comment is good and should stay, but should also be repeated in the section discussion Program.cs above (maybe even with an example line of code)

Reply:
done